app/views/comment_mailer/_comment_notification_footer_for_tag.html.erb

```diff
@@ -1,18 +1,15 @@
-<%= style_link("Read all comments on " + @comment.ultimate_parent.name, 
-               tag_comments_url(@comment.ultimate_parent)) %>
-<br />
-
 <% unless @noreply %>
-  <%= style_link("Reply to this comment",
-                 comment_url(@comment, add_comment_reply_id: @comment.id)) %>
+  <%= style_link(t(".reply.html"), comment_url(@comment, add_comment_reply_id: @comment.id)) %>
   <br />
 <% end %>
 
-<%= style_link("Go to the thread starting from this comment", 
-               comment_url(@comment)) %>
+<%= style_link(t(".comment_thread.html"), comment_url(@comment)) %>
 <br />
 
 <% unless @comment.id == @comment.thread %>
-  <%= style_link("Go to the thread to which this comment belongs", 
-                 comment_url(@comment.thread)) %>
+  <%= style_link(t(".parent_thread.html"), comment_url(@comment.thread)) %>
+  <br />
 <% end %>
+
+<%= style_link(t(".read_all_comments.html", tag: @comment.ultimate_parent.commentable_name),
```

Comment:
From what i can see, the tag name not being italics is now the only difference between the links here and the links in footer_for_other. Could we combine the two variants into one? Maybe we could only have the _other file for the extra stuff for non-tag comments and have these links here directly in the footer file? I'm sure you can figure out something good